test/opf.jl

```diff
@@ -564,13 +564,13 @@ end
         data = PMs.parse_file("../test/data/matpower/case14.m")
         pm = GenericPowerModel(data, SDPDecompForm)
 
-        cadj, lookup_index = PMs.chordal_extension(pm)
+        cadj, lookup_index, σ = PMs.chordal_extension(pm)
```

Comment:
Call me old fashioned but I prefer not to use unicode in PowerModels code.  Just use sigma instead.